pkg/kapp/cmd/app/deploy_flags.go

```diff
@@ -48,4 +49,5 @@ func (s *DeployFlags) Set(cmd *cobra.Command) {
 
 	cmd.Flags().BoolVar(&s.Logs, "logs", true, fmt.Sprintf("Show logs from Pods annotated as '%s'", deployLogsAnnKey))
 	cmd.Flags().BoolVar(&s.LogsAll, "logs-all", false, "Show logs from all Pods")
+	cmd.Flags().StringVar(&s.AppMetadataFile, "app-metadata-file", "", "write app metatadata to file")
```

Comment:
nit:
```suggestion
	cmd.Flags().StringVar(&s.AppMetadataFile, "app-metadata-file", "", "Write app metadata to file")
```

Also, maybe we would want to suggest that it accepts a filename as input? Something like "Set filename to write app metadata"?